packages/pyolite-kernel/py/pyolite/pyolite/__init__.py

```diff
@@ -15,7 +17,26 @@
 import pyolite
 from .kernel import Pyolite
 
-# TODO: until we have a proper display module
-sys.modules["IPython.display"] = pyolite.display
-
 kernel_instance = Pyolite()
+
+
+# TODO: Make an ipython module mock
+class IPMock():
+    def __init__(self, kernel):
+        self.kernel = kernel
+
+
+class InteractiveShellMock():
+    pass
+
+
+ip_mock = IPMock(kernel_instance)
+
+ip = types.ModuleType('IPython')
+ip.get_ipython = lambda: ip_mock
+ip.InteractiveShell = InteractiveShellMock
+
+sys.modules["IPython.display"] = pyolite.display
+sys.modules["IPython"] = ip
+sys.modules["IPython.core.getipython"] = ip
+sys.modules["IPython.core.interactiveshell"] = ip
```

Comment:
This needs to be removed. We should make a "real" IPython so that ipython does not get installed anymore

Reply:
Right. It's fine to leave it like this for now and we can track that separately.